src/config.json

```diff
@@ -2,6 +2,7 @@
   "version": 2,
   "tolerance": 2,
   "fuzzylist": [
+    "openmev.org",
```

Comment:
Could you move this to a separate PR? So that we can consider the functional changes separately from the config changes.

Reply:
sure, in fact I opened a PR specifically for this when I saw PR's being merged that only added to the list.

Reply:
It looks like there's still a config update in this PR. Also you've updated the package version and license. Please remove those changes as well.